compiler/optimizer/ValuePropagationCommon.cpp

```diff
@@ -1528,56 +1541,30 @@ void OMR::ValuePropagation::transformArrayCopyCall(TR::Node *node)
             }
          }
 
-      // -------------------------------------------------------------------
-      // Determine element size and array stride
-      // -------------------------------------------------------------------
-      if (isStringCompressedArrayCopy)
+      if (srcArrayLength)
```

Comment:
Previously this was guarded with a `if(elementSize)`. Is that check not needed anymore or was it never a necessary check?